tools/torchci/queue_alert.py

```diff
@@ -85,6 +121,17 @@ def gen_issue(queues: List[QueueInfo]) -> Any:
     return issue
 
 
+@lru_cache
+def get_queues() -> List[Dict[str, Any]]:
+    # %7B%7D = encoded {}
+    url = (
+        "https://hud.pytorch.org/api/clickhouse/queued_jobs_by_label?parameters=%7B%7D"
```

Comment:
Is this API continue to work?  I suspect that it might not given the bot protection that we have turned on and it will need the auth part like Dr.CI